Features/MyProfileLanguages.feature

```diff
@@ -0,0 +1,38 @@
+Feature: This test suite runs my profile language related test scenarios
+
+Scenario: 1. Verify user is able to add known languages with proficiency level
+	Given User logs into Mars application
+	When User adds 'English' language with 'Fluent' proficiency level into the known languages list
+	Then The 'English' language with 'Fluent' proficiency level should appear in the known languages list
+
+Scenario: 2. Verify user is able to edit an existing known language with proficiency level
```

Comment:
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/